dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/imports/TrackerImportControllerTest.java

```diff
@@ -224,6 +302,6 @@ void shouldReturnBadRequestWhenInvalidSkipRuleEngineIsPassed() {
         400,
         "ERROR",
         "Value 'INVALID' is not valid for parameter skipRuleEngine. It should be of type boolean",
-        POST("/tracker?async=false&skipRuleEngine=INVALID", "{}").content(HttpStatus.BAD_REQUEST));
```

Comment:
What was the previous vs current behaviour if async parameter was not provided?
I assume currently it defaults to async=false and takes the sync route. But previously?

Reply:
> What was the previous vs current behaviour if async parameter was not provided?

See all the HTTP examples with responses in the PR description 😋 The default behavior before and now is

No parameter => causes an async import

This is now easy to see with `ImportRequestParams.async = true` default.